.github/workflows/codespell.yml

```diff
@@ -11,4 +11,4 @@ jobs:
         skip: .git
         ignore_words_file: .codespellignore
         check_filenames: true
-        check_hidden: true
+        check_hidden: false
```

Comment:
I 've left it as `false` instead of removing it, so we can remember that it is disabled  by simply reading this file